core/templates/dev/head/pages/collection_player/CollectionPlayer.ts

```diff
@@ -52,12 +53,14 @@ oppia.controller('CollectionPlayer', [
   '$anchorScroll', '$http', '$location', '$scope',
   'AlertsService', 'CollectionObjectFactory',
   'CollectionPlaythroughObjectFactory', 'GuestCollectionProgressService',
+  'PageTitleService',
```

Comment:
Maybe you can fit `ReadOnlyCollectionBackendApiService` over here. (Following 80 chars max per line.)

Reply:
Done